library/ssl_tls13_client.c

```diff
@@ -2719,6 +2720,7 @@ static int ssl_encrypted_extensions_parse( mbedtls_ssl_context* ssl,
 
             default:
                 MBEDTLS_SSL_DEBUG_MSG( 3, ( "unknown extension found: %d ( ignoring )", ext_id ) );
+                break;
```

Comment:
Is this a convention? Since it is the last case, we don't have to add the break.

Reply:
I thought the convention is to have break for all cases in the `switch`. It is useful if some one move the `default` block up or add any new cases after that.

Reply:
In this file there are multiple occurrences of this pattern. I'd default to Hanno or others to provide guidance.